### PR TITLE
Feature#7 Design add place interest

### LINKED ISSUE #7: Design to add places of interest
# REQUIREMENTS
As a user, I want to see a screen that allows me to add the zip code and the name of the place where I want to make the report.

# DESIGN
image

### PR DESCRIPTION
**Pull request by YilKanda**
# Description
 - What?: A screen has been added that allows saving a new place of interest in the database.
 - Why?: This screen will allow us to save places of interest where citizen complaints can be made.
 
# Testing
It is important to install the necessary node libraries, from the command line:

`npm install`

While we are developing, from the command line we will be able to execute the command:

`npx react-native run-android`

To run our application on Android.

In the case that we have a simulator already established and open, the application will be launched in that simulator and we will be able to test it. 

If we want to test it on a real phone, we can connect our Android phone or iPhone to the computer, activate the developer options in the case of Android.

### Phone
imagen

imagen

### Log
imagen

imagen

### Cloud Firestore
imagen

### CODE REVIEW

File: DenunciaCiudadana/src/actions/placesActions.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import axios from 'axios';
+import firestore from '@react-native-firebase/firestore';
+import {types} from '../types/type';
+
+export const getPlacesAPI = value => {
+  const API_URL = 'https://api.copomex.com/query';
+  const API_METHOD = 'info_cp';
+  const API_SEARCH = value;
+  const API_PARAMS = 'type=simplified';
```
Comment (jkzasori) [inline DenunciaCiudadana/src/actions/placesActions.js]: I recommend to put all these constant in another file with name "service" or something like that to be more organized your files.

**Review by jkzasori — DISMISSED** [review]

File: DenunciaCiudadana/src/screens/AddPlacesScreen.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        buttonTitle={'Buscar Lugar de Interés en API'}
+        onPress={() => dispatch(getPlacesAPI(zipCode))}
+      />
+
+      <FormButton
+        buttonTitle={'Guardar'}
+        onPress={() => createPlace(zipCode, colony)}
+      />
+    </View>
+  );
```
Comment (fernandoamz) [inline DenunciaCiudadana/src/screens/AddPlacesScreen.js]: Move all the texts in a constants file.

**Review by fernandoamz — CHANGES_REQUESTED** [review]

File: DenunciaCiudadana/src/utils/ConstantAPI.js [hunk 3]
```diff
@@ -0,0 +1,6 @@
+export const ConstantAPI = {
+  API_URL: 'https://api.copomex.com/query',
+  API_METHOD: 'info_cp',
+  API_PARAMS: 'type=simplified',
+  API_KEY: '9ccb35bf-7bc8-467d-a57a-66ea01df0e51',
+};
```
Comment (fernandoamz) [inline DenunciaCiudadana/src/utils/ConstantAPI.js]: use a .env file to set this values

**Review by fernandoamz — CHANGES_REQUESTED** [review]

File: DenunciaCiudadana/src/config/language/en.js [hunk 4]
```diff
@@ -0,0 +1,7 @@
+export default {
+  ADD_PLACES_Title: 'Add Places of Interest',
+  ADD_PLACES_Postal_Code: 'Postal Code',
+  ADD_PLACES_Settlement: 'Settlement',
+  ADD_PLACES_Search: 'Search Place of Interest in API',
+  ADD_PLACES_Save: 'Save',
```
Comment (fernandoamz) [inline DenunciaCiudadana/src/config/language/en.js]:
```suggestion
  ADD_PLACES_TiITLE: 'Add Places of Interest',
  ADD_PLACES_POSTAL_CODE: 'Postal Code',
  ADD_PLACES_SETTLEMENT: 'Settlement',
  ADD_PLACES_SEARCH: 'Search Place of Interest in API',
  ADD_PLACES_SAVE: 'Save',
```

**Review by fernandoamz — CHANGES_REQUESTED** [review]

**Review by fernandoamz — APPROVED** [review]

File: DenunciaCiudadana/src/actions/placesActions.js [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+import firestore from '@react-native-firebase/firestore';
+import {types} from '../types/type';
+import {API_URL, API_METHOD, API_KEY} from '@env';
+
+export const getPlacesAPI = value => {
+  return async dispatch => {
+    const BASE_URL = `${API_URL}/${API_METHOD}/${value}?=type=simplified&token=${API_KEY}`;
+    try {
+      const res = await axios.get(`${BASE_URL}`);
+      console.log(res.data);
```
Comment (JacquelineFM) [inline DenunciaCiudadana/src/actions/placesActions.js:11]: If you don't need it, you could delete it, but everything else is great! :3

**Review by JacquelineFM — APPROVED** [review]

**Pull request merged**